docker/test/irods_client_rest_cpp.json

```diff
@@ -0,0 +1,85 @@
+{
+    "irods_client_environment": {
+        "zone": "tempZone",
+        "host": "irods-catalog-provider",
+        "port": 1247,
+        "ssl_negotiation_policy": "CS_NEG_REFUSE",
+        "rodsadmin_username": "rods",
+        "rodsadmin_password": "rods"
```

Comment:
Obviously, this whole section will have to change depending on where the iRODS server is. This provides a strong case for using a volume mount to inject this file when the service is stood up.